### DISCUSSION TITLE
Compile  Fairy stockfisk source

### DISCUSSION THREAD

**Discussion by uwb999** [fairy-stockfish/Fairy-Stockfish#467] [Q&A]
Hi all,

my name 's Tow.
I downloaded Fairy stockfisk source to compile them. I used Visual studio to compile them but it got error.
Could you show me how to compile source code with visual studio.

**Comment by ianfab** [comment]
Hi. Compiling with visual studio is not officially supported, but it might still work. The standard setup for compiling Fairy-Stockfish on Windows is with msys2/mingw as described in https://github.com/ianfab/Fairy-Stockfish/wiki/Compiling-Fairy-Stockfish and the links there. However, unless there is a specific reason why you want to compile it yourself you can also use the official releases https://github.com/ianfab/Fairy-Stockfish/releases or automatically built development versions https://github.com/ianfab/Fairy-Stockfish/actions/workflows/release.yml.

**Reply by uwb999** [reply]
thank you very much. I want to edit uci_elo fairy stockfish, but there aren't any gui can adjust engine. it still on default value 1350.

**Reply by uwb999** [reply]
i have just build fairy stockfish, but it hasn't been working.
Do you know this error. I built source code by mingw.
![ma nó cay](https://user-images.githubusercontent.com/103892428/165720338-61954211-5af7-4ee8-ab82-9e72d804f6f0.PNG)

**Comment by uwb999** [comment]
Hi guy, I have used VS version 2019 and used x64 platform.

Vào Th 4, 8 thg 6, 2022 vào lúc 02:33 movepick ***@***.***>
đã viết:
